src/config/defaults.ts

```diff
@@ -19,6 +19,14 @@ export interface FileOptions {
   readonly manifestFileName?: string;
 }
 
+export interface ShortcutOptions {
+  readonly name: string;
+  readonly short_name?: string;
+  readonly description?: string;
+  readonly url: string;
+  readonly icon?: string | string[];
```

Comment:
`Buffer` was mentioned in a comment. I suppose we may allow it too. 

`string | string[] | Buffer | Buffer[]` should be ok.

Reply:
Right, but the type of `manifestMaskable`contradicts the readme in that point too.
I've also fixed that.